code/modules/reagents/reagents/medicine.dm

```diff
@@ -25,7 +25,7 @@
 /datum/reagent/medicine/bicaridine
 	name = "Bicaridine"
 	id = "bicaridine"
-	description = "Bicaridine is an analgesic medication and can be used to treat blunt trauma."
+	description = "Bicaridine is an is analgesic medication and can be used to treat trauma at a moderate rate. Will eventually stop external bleeding. Less affective in Cht'mant."
```

Comment:
```suggestion
	description = "Bicaridine is an is analgesic medication and can be used to treat trauma at a moderate rate. Will lessen external bleeding. Less affective in Cht'mant."
```

Reply:
Every bleeding stops eventually, lel. Goddamnit, I wanted the analgesic out! _grumbles_

Reply:
blood cloting is a %, 0.1 = 10% less bleeding 1 = 100% less bleeding, ect ect
Its more accurate to put that it lessens it then "eventually stop" do to healing the wound